src/components/shared/Touchable.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import {
+  TouchableNativeFeedbackProps,
+  TouchableHighlightProps,
+  Platform,
+  TouchableNativeFeedback,
+  TouchableHighlight
+} from "react-native";
+
+type TouchableProps = TouchableNativeFeedbackProps | TouchableHighlightProps;
+
+const Touchable: React.FunctionComponent<TouchableProps> = ({
+  children,
+  ...props
+}) => {
+  if (Platform.OS === "android") {
+    return (
+      <TouchableNativeFeedback {...props}>{children}</TouchableNativeFeedback>
```

Comment:
How are you able to pass TouchableHighlightProps to TouchableNativeFeedback
and vice versa.
This should be throwing an error.